### ISSUE TITLE
Inconsistency between passing multiple pgp arguments into publish and publishAll commands

### ISSUE THREAD

**Issue by ghostbuster91** [com-lihaoyi/mill#1227]
Hi,

According to the docs it should be possible to pass multiple pgp arguments into mill's publish command: 
> Which you can then publish using the mill foo.publish command, which takes your sonatype credentials ( e.g. lihaoyi:foobarbaz) and GPG password as inputs:

When using `publishAll` command it can be done as follows:
```
./mill -j 8  mill.scalalib.PublishModule/publishAll __.publishArtifacts SONATYPE_USER:$SONATYPE_PASS \
 --gpgArgs --passphrase=$GPG_PASS,--batch,--yes,-a,-b --release true
```

However, such input doesn't work when trying to publish a single module using std syntax: 
```
./mill chessModel.publish $creds --release true --gpgArgs '-ab --yes'
```
```
gpg: invalid option "-ab --yes"
1 targets failed
chessModel.publish os.SubprocessException: CommandResult 2
```
(Other similar variation of the above approach also fails)

It seems that the only way to pass multiple pgpArguments into `publish` command is to use a different syntax:
```
./mill chessModel.publish $creds --release true --gpgArgs -ab --gpgArgs --yes
```

Gitter thread for more details: https://gitter.im/lihaoyi/mill?at=60490ce5a3a2f04f1fe87132

**Comment by lefou** [comment]
This should be fixed by #1270.  The following works after merging:

```
./mill chessModel.publish $creds --release true --gpgArgs -a,-b,--yes
```

**Comment by ghostbuster91** [comment]
Great, many thanks!
